eng/common/templates-official/jobs/source-build.yml

```diff
@@ -14,7 +14,7 @@ parameters:
   # This is the default platform provided by Arcade, intended for use by a managed-only repo.
   defaultManagedPlatform:
     name: 'Managed'
-    container: 'mcr.microsoft.com/dotnet-buildtools/prereqs:centos-7-3e800f1-20190501005343'
```

Comment:
These files come from Arcade & will be overwritten by the next Arcade update - I recommend making the updates in https://github.com/dotnet/arcade/blob/release/6.0/eng/common/templates/jobs/source-build.yml and https://github.com/dotnet/arcade/blob/release/6.0/eng/common/templates-official/jobs/source-build.yml